### ISSUE TITLE
New Patch, New Bugs

### ISSUE THREAD

**Issue by STRATZ-Ken** [skadistats/clarity#312]
```
skadistats.clarity.ClarityException: Entity not found for update at index 270. Entity update cannot be parsed!
    at skadistats.clarity.processor.entities.Entities.processPacketEntities(Entities.java:391)
    at skadistats.clarity.processor.entities.Entities.processAndRunPacketEntities(Entities.java:318)
    at skadistats.clarity.processor.entities.Entities.onPacketEntities(Entities.java:313)
    at skadistats.clarity.event.AbstractInvocationPoint.invoke(AbstractInvocationPoint.java:86)
    at skadistats.clarity.event.Event.raise(Event.java:41)
    at skadistats.clarity.processor.reader.InputSourceProcessor.processEmbedded(InputSourceProcessor.java:245)
    at skadistats.clarity.event.AbstractInvocationPoint.invoke(AbstractInvocationPoint.java:86)
    at skadistats.clarity.event.Event.raise(Event.java:41)
    at skadistats.clarity.processor.reader.InputSourceProcessor.processSource(InputSourceProcessor.java:172)
    at skadistats.clarity.event.AbstractInvocationPoint.invoke(AbstractInvocationPoint.java:86)
    at skadistats.clarity.event.Event.raise(Event.java:41)
    at skadistats.clarity.processor.runner.AbstractFileRunner.initAndRunWith(AbstractFileRunner.java:38)
    at skadistats.clarity.processor.runner.SimpleRunner.runWith(SimpleRunner.java:33)
    at Worker$1.handleDelivery(Worker.java:151)
    at com.rabbitmq.client.impl.ConsumerDispatcher$5.run(ConsumerDispatcher.java:149)
    at com.rabbitmq.client.impl.ConsumerWorkService$WorkPoolRunnable.run(ConsumerWorkService.java:111)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
    at java.lang.Thread.run(Thread.java:833)
```

**Comment by mcat1997** [comment]
me too

**Comment by spheenik** [comment]
Will try to fix over the weekend. Not possible earlier.

**Comment by spheenik** [comment]
@STRATZ-Ken please provide replay.

**Comment by p0d3x** [comment]
@spheenik we have similar issues, we'll provide some demos for you.

**Comment by STRATZ-Ken** [comment]
> @STRATZ-Ken please provide replay.

Every match.  Not a single match being parsed.

**Comment by howardchung** [comment]
example: http://replay122.valve.net/570/7750658331_1799511182.dem.bz2

**Comment by spheenik** [comment]
Released `3.0.5` with the fix.

**Comment by STRATZ-Ken** [comment]
Working.  Parsed about 2,000 matches already. Thanks so much!
